views/sections/SignIn/index.js

```diff
@@ -21,26 +27,90 @@ const SignIn = ({ progress, error, signIn }: Props) => {
     [signIn, email, password]
   )
 
+  useEffect(() => {
+    const _onInit = (auth2) => {
+      console.log('init OK', auth2)
+    }
+    const _onError = (err) => {
+      console.log('error', err)
+    }
+    window.gapi.load('auth2', () => {
+      window.gapi.auth2
+        .init({
+          client_id:
+            '172622657837-ac622rrsh2miq8din8h5muof9ki40927.apps.googleusercontent.com',
+        })
+        .then(_onInit, _onError)
+    })
+  })
+
+  const googleSignIn = (e) => {
+    // eslint-disable-next-line no-unused-expressions
+    e?.preventDefault()
+    const auth2 = window.gapi.auth2.getAuthInstance()
+    auth2.signIn().then((googleUser) => {
+      const profile = googleUser.getBasicProfile()
+      console.log(`ID: ${profile.getId()}`)
+      console.log(`Image URL: ${profile.getImageUrl()}`)
+      console.log(`Email: ${profile.getEmail()}`)
+      const { id_token } = googleUser.getAuthResponse()
+      console.log(`ID Token: ${id_token}`)
+    })
+  }
+
   return (
-    <section className="signIn">
-      <Header />
-      <form onSubmit={submit}>
-        <input
+    <section className="sign centered">
+      <Link href="/">
+        <span className="sign__logo">
+          <Logo />
+        </span>
+      </Link>
+      <form onSubmit={submit} className="sign__form aic">
+        <span className="sign__caption">Sign In</span>
+        <Input
           value={email}
-          type="text"
           placeholder="Email"
-          onChange={(e) => setEmail(e.target.value)}
+          handleChange={setEmail}
+          wrapClassName="sign__email"
         />
-        <input
+        <PasswordInput
           value={password}
-          type="text"
           placeholder="Password"
-          onChange={(e) => setPassword(e.target.value)}
+          handleChange={setPassword}
+          wrapClassName="sign__pass"
         />
-        <button type="submit" disabled={progress}>
+        <Button
+          type="submit"
+          disabled={progress}
+          height="lg"
+          className="sign__submit"
+        >
           Sign In
-        </button>
+        </Button>
+        <Button shape="text">Forgot a password?</Button>
+        <div className="sign__footer jcc">
+          <Button
+            className="sign__social"
+            shape="social"
+            handleClick={googleSignIn}
+          >
+            <Google className="sign__google" />
+            Google
+          </Button>
+          <Button className="sign__social" shape="social">
+            <span>
+              <Facebook className="sign__facebook" />
+            </span>
+            Facebook
+          </Button>
+        </div>
       </form>
+      <span className="sign__acc-info">
+        No account yet?
+        <Link href="/signUp" as="/sign-up">
+          <span className="sign__link">Sign Up</span>
+        </Link>
+      </span>
       <span className="error">{error && error}</span>
```

Comment:
We need to handle errors somehow. We will use notifications, right? So we can remove this line